pkg/sql/sem/tree/schedule.go

```diff
@@ -16,15 +16,20 @@ type FullBackupClause struct {
 	Recurrence Expr
 }
 
+type ScheduleLabelSpec struct {
+	IfNotExists bool
```

Comment:
I know there were some comments around the naming... and the confusion.
I would personally prefer `RequireUnique` field -- I think it conveys what this label means. You can describe that require unique is set when creating *new* schedules -- and that, as the name implies, it requires the label to be unique.

I don't feel too strongly about this though -- so if you think IfNotExists is a better name, then keep it.